docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    out = sinkhorn(PointCloud(x, y, epsilon=varep),
</pre><p>any reason why default parameters in Sinkhorn are not used here? rather than <code>varep</code> using <code>epsilon</code>  to call the parameter would be easier to parse.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>